docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -72,12 +72,12 @@ space. While the exact amount resource usage of each additional field depends
 on the type of field, an additional 1024B in memory use  for each additional
 mapped field on data nodes is a good approximation in most cases.
 As a result, the more mapped fields you have per index, the fewer
-indices and shards will each individual data node be able to hold,
+indices and shards each individual data node will be able to hold,
 all other things being equal. In addition to the per field overhead you should
 at least have an additional 512MB of heap available per data node.
 For example, holding 1000 indices that each contain 4000 fields will require
```

Comment:
I think it's better to talk about a node holding 1000 shards not indices here for calculating heap usage?

```suggestion
For example, holding 1000 shards that each contain 4000 fields will require
```

Reply:
How about something like `holding shards from 1000 different indices`? Mappers are shared across shards from the same index so adding multiple shards from each index doesn't add to this memory usage.

Reply:
Is there not also a per shard overhead per field (in lucene)?

Reply:
> Is there not also a per shard overhead per field (in lucene)?

it's relatively small compared to the ES overhead for beats indices. Given that we now use a value of 1KB here that gives us a lot of breathing space I think we more than cover the Lucene overheads in most cases. I'd rather keep the per index documentation here as it applies to the vast majority of cases (and as far as I can tell at least for the Beats mappings, would even apply well for significant outliers with say 5 shards per the same index on a data node).